buildSrc/src/main/java/FileExt.kt

```diff
@@ -0,0 +1,12 @@
+import java.io.File
+import java.util.*
+
+fun File.loadGradleProperties(fileName: String): Properties {
```

Comment:
How about renaming it to `loadProperties` since it can load any properties file with the supplied name?
```suggestion
fun File.loadProperties(fileName: String): Properties {
```